proxy/src/console/provider/neon.rs

```diff
@@ -88,7 +89,9 @@ impl Api {
                 .build()?;
 
             info!(url = request.url().as_str(), "sending http request");
+            let start = Instant::now();
             let response = self.endpoint.execute(request).await?;
+            trace!(duration = ?start.elapsed(), "received http response");
```

Comment:
Maybe `info!` level will be better? It will make grep across all staging/production logs easier, as we don't need to change log level manually everywhere.

Reply:
... and I don't think we enabled trace in production?

Reply:
I guess, I explicitly chose trace because I didn't want to spam too much in prod, but if you think it's ok then sure